engine/session.js

```diff
@@ -1123,6 +1178,8 @@ class Session {
             sessionState.currentVod = await this._sessionState.setCurrentVod(currentVod, { ttl: currentVod.getDuration() * 1000 });
             this.leaderIsSettingNextVod = false;
             await this._playheadState.set("playheadRef", Date.now());
+            await this._playheadState.set("diffCompensation", this.diffCompensation);
+            debug(`[${this._sessionId}]: sharing durrent vods diffCompensation=${this.diffCompensation}`);
```

Comment:
Adding this to store, so that follower nodes can also calculate a propper tick interval. 
Followers read this from store at the same time they read a new currentVod.